lib/MultiColumnList/MCLRenderer.js

```diff
@@ -1751,7 +1751,7 @@ class MCLRenderer extends React.Component {
       const scrollBarHeight = 20;
       newHeight = Math.max(
         ((pagingType !== pagingTypes.SCROLL ? receivedRows : totalCount || receivedRows) * minimumRowHeight),
-        (height - this.headerHeight - scrollBarHeight)
+        (height - this.headerHeight - scrollBarHeight - this.paginationHeight)
```

Comment:
`this.paginationHeight` will be 40 even if `pagingType !== pagingTypes.PREV_NEXT`. Let's utilize this:

`const paginationHeight = pagingType === pagingTypes.PREV_NEXT ? this.paginationHeight : 0;`
```suggestion
        (height - this.headerHeight - scrollBarHeight - paginationHeight)
```